services/remote-validator/src/test/java/tech/pegasys/teku/services/remotevalidator/RemoteValidatorApiTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.services.remotevalidator;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.common.primitives.UnsignedLong;
+import io.javalin.http.Context;
+import io.javalin.websocket.WsConnectContext;
+import java.io.IOException;
+import java.util.UUID;
+import java.util.function.Consumer;
+import javax.servlet.http.HttpServletRequest;
+import org.eclipse.jetty.websocket.api.RemoteEndpoint;
+import org.eclipse.jetty.websocket.api.Session;
+import org.eclipse.jetty.websocket.api.StatusCode;
+import org.eclipse.jetty.websocket.servlet.ServletUpgradeRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import tech.pegasys.teku.services.remotevalidator.RemoteValidatorSubscriptions.SubscriptionStatus;
+import tech.pegasys.teku.util.config.TekuConfiguration;
+
+@ExtendWith(MockitoExtension.class)
+class RemoteValidatorApiTest {
+
+  @Mock private TekuConfiguration configuration;
```

Comment:
```suggestion
  private TekuConfiguration configuration = mock(TekuConfiguration.class);
```

and so on. for the other fields. :)